pkgs/development/tools/supabase-cli/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildGo118Module
+, fetchFromGitHub
+, installShellFiles
+}:
+
+buildGo118Module rec {
+  pname = "supabase-cli";
+  version = "1.22.1";
+
+  src = fetchFromGitHub {
+    owner = "supabase";
+    repo = "cli";
+    rev = "v${version}";
+    sha256 = "sha256-lQA6wWwfVfOcJqI1XdOuKnMk0hPMiMPEIa8Kb/2mvlI=";
+  };
+
+  vendorSha256 = "sha256-6yEQkHtMz6YBhWSE2ga+DBb+m5HuCdMuJYmxhS59lnw=";
+
+  ldflags = [ "-X" "github.com/supabase/cli/cmd.version=${version}" ];
+
+  doCheck = false; # tests are trying to connect to localhost
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    rm ${placeholder "out"}/bin/{codegen,docgen,listdep}
+    mv ${placeholder "out"}/bin/{cli,supabase}
```

Comment:
The name cli seems like a pretty overloaded name for in the PATH...

Reply:
Yes, that is why I renamed it to `supabase`, this is also the recommended name by upstream.